### PR TITLE
Realised that there were a few more bits (special params) that were not yet documented.

### PR DESCRIPTION
**Pull request by ewels**
Hopefully this is the last of them.

### CODE REVIEW

File: docs/parameters/validation.md [hunk 1]
```diff
@@ -114,6 +114,17 @@ params.validationFailUnrecognisedParams = true
         --8<-- "examples/validationFailUnrecognisedParams/pipeline/main.nf"
         ```
 
+## Ignoring unrecognized parameters
+
+Sometimes, a parameter that you want to set may not be described in the pipeline schema for a good reason.
+Maybe it's something you're using in your Nextflow configuration setup for your compute environment,
+or it's a complex parameter that cannot be handled in the schema.
```
Comment (sateeshperi) [inline docs/parameters/validation.md:121]: could you give an example of such complex unrecognized params?

File: docs/parameters/validation.md [hunk 2]
```diff
@@ -114,6 +114,17 @@ params.validationFailUnrecognisedParams = true
...
+
+In these cases, to avoid getting warnings when that unrecognised parameter is set,
+you can use `--validationSchemaIgnoreParams` / `params.validationSchemaIgnoreParams`.
```
Comment (sateeshperi) [inline docs/parameters/validation.md:124]:
```suggestion
you can use `--validationSchemaIgnoreParams` / `params.validationSchemaIgnoreParams = true`.
```
?

File: docs/parameters/summary_log.md [hunk 3]
```diff
@@ -8,7 +8,13 @@ description: Return a summary of the parameters being set.
...
 This function returns a string that can be logged to the terminal, summarizing the parameters provided to the pipeline.
 
-It only returns the provided parameters that are **different** to the schema default values.
+!!! note
+
+    The summary only returns the provided parameters that are **different** to the schema default values.
+
+    This is because pipelines often have a very long list of parameters.
+    If we print all of them, it's difficult to see which ones were customised.
+    That's what users usually care about: for example, did I make a typo, was my config resolved properly, etc.
```
Comment (sateeshperi) [inline docs/parameters/summary_log.md:17]:
```suggestion
The summary prioritizes displaying only the parameters that deviate from the default schema values. This is to streamline the extensive parameter lists often associated with pipelines, and highlight the customized elements. This feature is essential for users to verify their configurations, like checking for typos or confirming proper resolution, without wading through an array of default settings.
```

File: docs/parameters/validation.md (hunk 1 above)
Comment (ewels) [reply docs/parameters/validation.md:121]: I was thinking of nested params. These have their own section, so can link to that.

**Review by nvnieuwk — APPROVED** [review]
LGTM if the comments of @sateeshperi are addressed :)

File: docs/parameters/validation.md (hunk 2 above)
Comment (mirpedrol) [reply docs/parameters/validation.md:124]: This parameter should be a list of the parameters to ignore, it's explained in the next line

**Review by mirpedrol — APPROVED** [review]
LGTM

File: docs/parameters/validation.md (hunk 1 above)
Comment (mirpedrol) [reply docs/parameters/validation.md:121]:
```suggestion
or it's a complex parameter that cannot be handled in the schema, such as [nested parameters](../nextflow_schema/nextflow_schema_specification.md#nested-parameters).
```

File: docs/parameters/summary_log.md (hunk 3 above)
Comment (ewels) [reply docs/parameters/summary_log.md]: @mirpedrol did you check the rendering on this before applying? I think the indentation was lost, which will have broken the admonition box.

Comment (mirpedrol) [reply docs/parameters/summary_log.md]: I didn't, will fix it! 😄

**Pull request merged**